### DISCUSSION TITLE
How To Integate New MindMap Application To My Project

### DISCUSSION THREAD

**Discussion by QuangLanh** [clientIO/joint#2756] [Q&A]
### Introduction

Hello,

I noticed that the latest versions of JointJS have a new Mindmap application. Currently, I want to integrate this application into our project, but I cannot find any documentation or examples on how to integrate and configure this application into another project. Could you please provide me with an example to integrate into other project?
Does this app have the ability to add, delete and drag and drop components and lines?
![image](https://github.com/user-attachments/assets/7c24a82f-c809-4a9b-a150-7b86c8e6fd48)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
I think it's a duplicate of https://github.com/clientIO/joint/discussions/2757.

**Reply by QuangLanh** [reply]
What I mean here is that I want to build the mindmap project into an sdk or a package to export the api for using.

**Reply by kumilingus** [reply]
In the MindMap example, there is an `App` class that you can instantiate and use in another project.
You can also extend the class with your own methods (API for controlling the application from outside).

```ts
const app = new App(document.getElementById('app'));
app.start();
```
